### DISCUSSION TITLE
additional info on reader.pages

### DISCUSSION THREAD

**Discussion by keyuchen21** [py-pdf/pypdf#2658] [Q&A]
- I'm wondering what's the format(not sure if it is the correct words here) of the reader.pages output?
- what format does it specifically followed? 

from pypdf import PdfReader

# Load the PDF
reader = PdfReader("GeoBase_NHNC1_Data_Model_UML_EN.pdf")

# Access a specific page
page = reader.pages[3]

**Comment by stefan6419846** [comment]
I recommend you to have a look at the corresponding type hints which state that this returns a list of `PageObject`s, which wrap each page into a Python class accordingly.
